src/MongoDB.Driver.Core/Core/Compression/Zstd/Zstandard64NativeMethods.cs

```diff
@@ -0,0 +1,241 @@
+﻿/* Copyright 2020–present MongoDB Inc.
```

Comment:
file renaming is one of the most annoying things in github.. Now previos comments are lost..